feature-libs/customer-ticketing/components/shared/customer-ticketing-dialog/customer-ticketing-dialog.component.spec.ts

```diff
@@ -13,6 +14,7 @@ describe('CustomerTicketingDialogComponent', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
+      imports: [I18nTestingModule],
       declarations: [CustomerTicketingDialogComponent],
     }).compileComponents();
```

Comment:
Is there a dedicated ticket for unit tests here? I thought it would be done once the apis will be integrated in conjunction with the template that was created a while back

Reply:
Applies to all specs here so I don't flag every single one of them